Frameworks/UIKit/UINibUnarchiver.mm

```diff
@@ -203,7 +203,13 @@ static id constructObject(UINibUnarchiver* self, Object* pObj) {
         id classId = pObj->classType; // objc_getClass(pObj->className);
         assert(classId != nil);
 
-        pObj->cachedId = [classId alloc];
+        // TODO: #1105 alloc without init is invalid, and not all classes support alloc-init anyway
+        // look for a proper fix to this issue, rather than just special-casing for UIFont
+        if ([classId isEqual:[UIFont class]]) {
+            pObj->cachedId = [[UIFont defaultFont] retain];
```

Comment:
Hmm, if it's the case that the UIFont still has to support initWithCoder despite of its doc (was not previously aware of this, and thus current CT-DW UIFont doesn't), then I can just change up UIFont to support alloc and initWithCoder and we won't need the special case here.

---
In reply to: [82917523](https://github.com/Microsoft/WinObjC/pull/1106#discussion_r82917523) [](ancestors = 82917523)